### PR TITLE
Update language identifiers and styleguide.

### LINKED ISSUE #1737: Replace language identifier to use the supported ones for OPS
We need to add the list of supported language to our [style guide](https://github.com/dotnet/docs/blob/master/styleguide/template.md#inline-code-blocks-with-language-identifier) and replace the alternative ones currently in use in our repo to the officially supported ones, since new features will be done based on those.

For example, replace \`\`\`cs with \`\`\`csharp, \`\`\`fs with \`\`\`fsharp, etc.

The list of supported identifiers can be found in this internal site:
https://review.docs.microsoft.com/en-us/csi/msdocs/code?branch=master

/cc @tompratt-AQ

### PR DESCRIPTION
**Pull request by tompratt-AQ**
# Title
Update language identifiers and style guide for supported languages.

## Summary
Added a list of supported languages with language IDs and markdown labels to the style guide. Fixed language identifiers for C#, F# and PHP in all files to match the supported markdown labels.

Fixes #1737 

## Suggested Reviewers
@mairaw

### CODE REVIEW

File: docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The key concept to understand is that an async expression’s type is `Async<'T>
 
 For example, say you wanted to download the HTML from dotnetfoundation.org without blocking the main thread. You can accomplish it like this:
 
-```fs
+```fsharpharp
```
Comment (mairaw) [inline docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md]: should be fs only. maybe search your working branch to see if you have other instances of fsharpharp or csharpharp

File: docs/csharp/programming-guide/classes-and-structs/static-classes-and-static-class-members.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ translation.priority.ht:
 # Static Classes and Static Class Members (C# Programming Guide)
 A [static](../../../csharp/language-reference/keywords/static.md) class is basically the same as a non-static class, but there is one difference: a static class cannot be instantiated. In other words, you cannot use the [new](../../../csharp/language-reference/keywords/new.md) keyword to create a variable of the class type. Because there is no instance variable, you access the members of a static class by using the class name itself. For example, if you have a static class that is named `UtilityClass` that has a public method named `MethodA`, you call the method as shown in the following example:  
   
-```cs  
+```csharp  
 UtilityClass.MethodA();  
 ```
```
Comment (mairaw) [inline docs/csharp/programming-guide/classes-and-structs/static-classes-and-static-class-members.md]: the snippet below this one is missing the identifier. please add.

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks great @tompratt-AQ. Left a few comments.

File: docs/csharp/programming-guide/classes-and-structs/static-classes-and-static-class-members.md (hunk 2 above)
Comment (tompratt-AQ) [inline docs/csharp/programming-guide/classes-and-structs/static-classes-and-static-class-members.md]: I fixed these two issues and did not find any other instances like these.

**Review by mairaw — APPROVED** [review]

**Pull request merged**